### PR TITLE
added ancla helper package for repos that don't use uber/fx

### PR DESCRIPTION
**Pull request by maurafortino**
What's Included:
- https://github.com/xmidt-org/ancla/issues

### CODE REVIEW

**Review by denopink — APPROVED** [review]
lgtm! 🍻

File: anclaHelper.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package ancla
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	// nolint:staticcheck
+	"github.com/xmidt-org/webpa-common/v2/xmetrics"
```
Comment (denopink) [inline anclaHelper.go:6]: @maurafortino do we need to use "github.com/xmidt-org/webpa-common/v2/xmetrics"?

Comment (johnabass) [reply anclaHelper.go:6]: If `ancla` is already using `webpa-common` for metrics, this is fine for now.  We should move to `touchstone` ASAP, though.

If `ancla` doesn't have metrics, then we should use `touchstone` here.

**Comment by maurafortino** [comment]
Moving ancla helper package into caduceus so that we aren't introducing webpa-common into ancla.

**Pull request closed without merging**